oe_authentication.post_update.php

```diff
@@ -61,3 +61,12 @@ function oe_authentication_post_update_00006(): void {
     ->set('message_login_2fa_required', 'Your account is required to log in using a two-factor authentication method. Please <a href=":login">log in again via this link</a>.')
     ->save();
 }
+
+/**
+ * Add missing langcode.
+ */
+function oe_authentication_post_update_00007(): void {
+  \Drupal::configFactory()->getEditable('oe_authentication.settings')
+    ->set('langcode', 'en')
+    ->save();
+}
```

Comment:
This change makes sense to me.\
But, I have not found similar update hooks in core (I searched for `->set('langcode', 'en')`.\
So, what do we miss if we don't do this?

Also, is this a case for hook_update_N() or rather hook_post_update_NAME()?

Going to think about this or ask others..

Reply:
One thing I was wondering: Do we need to worry about sites that already saved this with e.g. `langcode: fr`?

I don't think this is a problem, because:
- The addition of the text value is very recent.
- The locale module does add the site default language to a config record that has translatable strings and no langcode. But this is actually something we want to fix, because we want 'en' here.